gke-usage-metering/src/data-studio.d.ts

```diff
@@ -0,0 +1,299 @@
+// Type definitions for Google Apps Script 2018-11-07
```

Comment:
You should be able to remove this file & use these types from npm directly.

To install the types dependency.
```
npm install --save-dev @types/google-apps-script
```

And make sure that your tsconfig has a line to look for the types see [the npm-downloads-ts example]

[the npm-downloads-ts example]: https://github.com/googledatastudio/example-connectors/blob/master/npm-downloads-ts/tsconfig.json#L5

Reply:
Done.